tests/acceptance/features/apiOcm/acceptInvitation.feature

```diff
@@ -1,4 +1,4 @@
-@ocm
+@ocm @notification
```

Comment:
is the tag required? and why?

Reply:
Then we should think of better solution again. Maybe do mailbox cleanup in the beforeScenario hook of scenarios that have `@email` tag?

CC @amrita-shrestha

Reply:
This should be done in separate PR